org.coolero.Coolero.yaml

```diff
@@ -0,0 +1,117 @@
+app-id: org.coolero.Coolero
+runtime: org.kde.Platform
+runtime-version: '6.2'
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm12
+command: coolero.sh
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=all
+  - --device=dri
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=org.freedesktop.Flatpak
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - '*.la'
+  - '*.a'
+modules:
+  - name: libusb
+    sources:
+      - type: archive
+        url: https://github.com/libusb/libusb/archive/v1.0.24.tar.gz
+        sha256: b7724c272dfc5713dce88ff717efd60f021ca5b7c8e30f08ebb2c42d2eea08ae
+
+  # prerequisites not handled by script
+  - name: cppy
+    buildsystem: simple
+    build-commands:
+      - pip3 install --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} cppy
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/36/64/be592e84c443a70ea5dcfb1c30bfe6f10ba7d8eb05c2264c7ceca8498548/cppy-1.1.0.tar.gz
+        sha256: 4eda6f1952054a270f32dc11df7c5e24b259a09fddf7bfaa5f33df9fb4a29642
+  - name: pyamdgpuinfo
+    buildsystem: simple
+    build-commands:
+      - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}"
+        --prefix=${FLATPAK_DEST} ./ --no-build-isolation
+    sources:
+      - type: git
+        url: https://github.com/mark9064/pyamdgpuinfo.git
+        tag: v2.1.1
+        commit: 06284da07ca5abf7756f247c72b0192b7d0c5b98
+      - type: shell
+        commands:
+          - sed -i 's|/usr/include/libdrm|/app/include/libdrm|' setup.py
+    modules:
+      - name: libdrm
+        buildsystem: simple
+        build-commands:
+          - meson build --prefix=/app
+          - ninja -C build/ install
+        sources:
+          - type: git
+            url: git://anongit.freedesktop.org/mesa/drm
+            branch: master
```

Comment:
please specific a specific revision or tag. NOT a branch.

Reply:
:+1:  updated to https release tag